applications/northstar/main.tf

```diff
@@ -91,6 +91,7 @@ locals {
   feature_config_vars = {
     DS_ENABLE_PASSWORD_GRANT = false
     DS_ENABLE_RATE_LIMITING  = true
+    DS_ENABLE_CUSTOMER_IO    = var.environment != "development"
```

Comment:
Can you remind me of how this works? This will just set it up in Heroku for us, but we can still edit it via Heroku?

Reply:
We will still be able to edit it via Heroku, but we probably won't want to outside of an emergency. Instead, we'd want to let Terraform manage these environment variables, both to set them initially & to "reset" them if they're edited by hand (so we can avoid [configuration drift](https://www.hashicorp.com/resources/how-can-i-prevent-configuration-drift)).

Reply:
Awesome!